content/departments/cloud/technical-docs/v1.1/mi1-1_onprem_data_migration.md

```diff
@@ -0,0 +1,207 @@
+# MI v1.1 on-prem data migration
+
+> NOTE: This process is still a work-in-progress. For more details, see [RFC 760](https://docs.google.com/document/d/1IAgXmv2TbtU_rWXtph-KFc3qbqswREIAxO1rGALuoMQ/edit#) and the [tracking issue](https://github.com/sourcegraph/customer/issues/1525), or cc @bobheadxi (Robert Lin).
+
+This process describes the current state of how to do a full data migration of an on-prem instance to a MI v1.1 Cloud instance.
+
+## Requirements
+
+The customer must:
+
+- have a Sourcegraph instance on v3.20.0 or later
+- use databases on Postgres 11 or later
+- _not_ have [on-disk database encryption](https://docs.sourcegraph.com/admin/config/encryption) enabled
+- have the latest version of [`src`](https://github.com/sourcegraph/src-cli)
```

Comment:
nit: "latest" is vague and hard to ensure... would "version same as or newer than the Sourcegraph instance" work/make sense?

Reply:
> would "version same as or newer than the Sourcegraph instance" work/make sense?

No - the tooling is in active development, so here "latest" means "latest" in the absolute sense (i.e. the most recently published `src` release). I think this is okay, since I anticipate the process will be very hand-holdy so we can actively ensure an admin has the most recent patches

Reply:
I'll note that this should be the latest release